### PR TITLE
Included option for SureSelectv7 analysis 

### PR DESCRIPTION
**Pull request by melferink**
* added SureSelectV7 option to run_nextflow_wes.sh
*modified WES.config and README

### CODE REVIEW

File: run_nextflow_wes.sh [hunk 1]
```diff
@@ -7,6 +7,16 @@ workflow_path='/hpc/diaggen/software/production/DxNextflowWES'
 input=`realpath -e $1`
 output=`realpath $2`
 email=$3
+
+design=${4-default}
+gatk_hc_interval_list='Tracks/ENSEMBL_UCSC_merged_collapsed_sorted_v3_CREv2_100bpflank.interval_list'
+picard_bait='Tracks/SureSelect_CREv2_elidS30409818_Covered.list'
```
Comment (ellendejong) [inline run_nextflow_wes.sh:13]: I believe these two lines can be removed and the default from WES.config can be used.
https://www.nextflow.io/docs/latest/config.html?highlight=params%20file#configuration-file 

You might need to change lines 47+48.

File: run_nextflow_wes.sh [hunk 2]
```diff
@@ -34,6 +44,8 @@ module load Java/1.8.0_60
 --fastq_path $input \
 --outdir $output \
 --email $email \
+--gatk_hc_interval_list $gatk_hc_interval_list \
+--picard_bait $picard_bait \
```
Comment (ellendejong) [inline run_nextflow_wes.sh:48]: I would prefer to see this replaced with something similar to:

${​​​​​​​gatk_hc_interval_list:-""}​​​​​​​ \
${picard_bait:-""} \

File: run_nextflow_wes.sh [hunk 3]
```diff
@@ -7,6 +7,16 @@ workflow_path='/hpc/diaggen/software/production/DxNextflowWES'
...
 output=`realpath $2`
 email=$3
+
+design=${4-default}
+gatk_hc_interval_list='Tracks/ENSEMBL_UCSC_merged_collapsed_sorted_v3_CREv2_100bpflank.interval_list'
+picard_bait='Tracks/SureSelect_CREv2_elidS30409818_Covered.list'
+if [ $design == "SSv7" ]; then
+    echo " #### SSv7 design target files are used for the analysis ####"
+    gatk_hc_interval_list='Tracks/ENSEMBL_UCSC_merged_collapsed_sorted_v3_CREv2_SSv7_100bpflank.interval_list'
+    picard_bait='Tracks/SureSelect_v7_elidS31285117_Covered.list'
```
Comment (ellendejong) [inline run_nextflow_wes.sh:17]: Example in next comment will work, if you apply these changes:
```
 gatk_hc_interval_list='--gatk_hc_interval_list Tracks/ENSEMBL_UCSC_merged_collapsed_sorted_v3_CREv2_SSv7_100bpflank.interval_list'
picard_bait='--picard_bait Tracks/SureSelect_v7_elidS31285117_Covered.list'
```

Comment (melferink) [reply run_nextflow_wes.sh]: That is... elegant!

**Review by ellendejong — APPROVED** [review]
No additional comments. Approved.

**Pull request merged**